private-templates-service/adaptivecards-templating-client/swagger.yaml

```diff
@@ -356,7 +349,7 @@ definitions:
       _id: 
         type: string
       json: 
-        type: string
+        type: object
```

Comment:
Line 430: should type should be object as well?

Reply:
For the authId?

Reply:
My bad, thought those were templates
![image](https://user-images.githubusercontent.com/40618774/76986074-6abd6380-68fe-11ea-8bb6-991571eb4d0a.png)

Reply:
Oh okay I see for the recently edited/viewed templates? I'll put up a separate PR to fix this, might need to also edit the recent calls